src/main/java/com/github/magrifle/data/searchapi/aspect/DataSearchApi.java

```diff
@@ -88,7 +88,8 @@ private Object buildSpecificationForMethod(ProceedingJoinPoint joinPoint, Search
                     matcher.group(2),
                     matcher.group(4),
                     matcher.group(3),
-                    matcher.group(5)
+                    matcher.group(5),
+                    searchApi.caseSensitive()
```

Comment:
could this be passed as a class field instead of through this method since it's going to be the same for every instance of the SpecificationsBuilder?

Reply:
Done